TenureInformationApi/serverless.yml

```diff
@@ -112,6 +116,22 @@ resources:
                   Action:
                     - "lambda:InvokeFunction"
                   Resource: "*"
+          - PolicyName: dynamo-stream
+            PolicyDocument:
+              Version: '2012-10-17'
+              Statement:
+                - Effect: Allow
+                  Action:
+                    - "iam:DescribeStream"
+                    - "iam:ListStreams"
+                    - "iam:GetRecords"  
+                    - "iam:GetShardIterator"
+                    - "dynamodb:DescribeStream"
+                    - "dynamodb:ListStreams"
+                    - "dynamodb:GetRecords"
+                    - "dynamodb:GetShardIterator"
+
+                  Resource: "*"
```

Comment:
This wildcard is temporary for the purposes of proof of concept.
It was agreed that once the real lambda gets created, the dynamo permissions will be set against the stream: `arn:aws:dynamodb:eu-west-2:364864573329:table/TenureInformation/stream/*`